### DISCUSSION TITLE
Can not connect to redis

### DISCUSSION THREAD

**Discussion by jaydond47** [Grokzen/docker-redis-cluster#142] [Q&A] [answered]
I cannot connect to redis when run  command : redis-cli -c -p 7000
Error : 
Could not connect to Redis at 127.0.0.1:7000: Connection refused

my docker-compose.yml file:

version: "3"
services:
  rediscluster:
    image: grokzen/redis-cluster
    environment:
      IP: 127.0.0.1
      CLUSTER_ONLY: 'true'
      MASTERS: 3
      SLAVES_PER_MASTER: 1
    ports:
      - '7000-7007:7000-7007'

OS infor : 
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

**Comment by Grokzen** [comment, marked as answer]
@trantiendung20101999 you must run the image in network mode: host for that to work. Look it up in the docker compose documentation how to modify your compose file for that to be enabled.
